lib/data/network/contact/tom_contact_api.dart

```diff
@@ -3,19 +3,13 @@ import 'package:fluffychat/data/network/identity_endpoint.dart';
 import 'package:fluffychat/di/global/get_it_initializer.dart';
 import 'package:fluffychat/di/global/network_di.dart';
 import 'package:fluffychat/domain/model/contact/contact_query.dart';
-import 'package:fluffychat/domain/model/contact/hash_details_response.dart';
-import 'package:fluffychat/domain/model/contact/lookup_list_mxid_request.dart';
-import 'package:fluffychat/domain/model/contact/lookup_list_mxid_response.dart';
 import 'package:fluffychat/domain/model/contact/lookup_mxid_request.dart';
 import 'package:fluffychat/domain/model/contact/lookup_mxid_response.dart';
 
 class TomContactAPI {
-  final DioClient _identityClient =
+  final DioClient _client =
```

Comment:
this client should use `tom` dio instance

Reply:
https://github.com/linagora/twake-on-matrix/issues/978

Reply:
noted, will fix it later depend on your issue #978

Reply:
thank you